### PR TITLE
docs: Implement Sidebar categories and Update guides content

### LINKED ISSUE #2229: docs: add integration Groq API
**Pages**
- Page(s) that need to be done
Regarding issue #2227 

Integration Docs for Groq API
Slug: /guides/integration/groq/

Resources:

Website: https://groq.com/
API Access: https://wow.groq.com/
image

**Success Criteria**
Content that should be covered

Helps user integrate Groq API to Jan 

**Additional context**
Examples, reference pages, resources

### PR DESCRIPTION
**Pull request by aindrajaya**
## Sidebar categories and Update guides content
- Add FAQs content
- Add Groq integration content
- Update Sidebar categories

## Fixes Issues

- Closes #2229 

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [x] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: docs/sidebars.js [hunk 1]
```diff
@@ -150,10 +150,79 @@ const sidebars = {
       ],
     },
   ],
+  // guidesSidebar: [
+  //   {
+  //     type: "autogenerated",
+  //     dirName: "guides",
+  //   },
+  // ],
   guidesSidebar: [
```
Comment (hieu-jan) [inline docs/sidebars.js:159]: For guidesSidebar: I think we should keep as current method autogenerated. 

Current change made sidebar broken, not good UX/ UI

image

File: docs/docs/guides/integration/groq.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  "object": "model",
+  "name": "Groq Integration",
+  "version": "1.0",
+  "description": "Integration with Groq API for enhanced functionality.",
+  "format": "api",
+  "sources": [],
+  "settings": {},
+  "parameters": {},
+  "metadata": {
+    "author": "Your Name",
```
Comment (hieu-jan) [inline docs/docs/guides/integration/groq.mdx:65]: Author should set based on model https://console.groq.com/docs/models. `Meta` for Llama2 and `Mistral`  for mixtral-8x7b-32768

File: docs/docs/guides/integration/groq.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    "tags": ["Groq Integration"]
+  },
+  "engine": "groq"
```
Comment (hieu-jan) [inline docs/docs/guides/integration/groq.mdx:68]: engine should be `openai`

File: docs/docs/guides/faq.mdx [hunk 4]
```diff
@@ -17,6 +17,80 @@ keywords:
...
+- **How to use GPU AMD for Jan?**
+  - Jan now supports AMD GPUs through Vulkan. This enhancement allows users with AMD graphics cards to leverage GPU acceleration, improving performance for AI model computations.
+
+- **Is Jan available for Android or iOS?**
+  - Jan is primarily focused on the Desktop app and does not currently offer mobile apps for Android or iOS. The development team is concentrating on enhancing the desktop experience.
+
+## Development and Features
+
+- **Can I use local files from my computer with Jan?**
+  - The feature to use local files is currently in development. For more information, please our offical [Discord](https://discord.com/invite/FTk2MvZwJH) for the latest update.
```
Comment (hieu-jan) [inline docs/docs/guides/faq.mdx:79]: Typo `please our official`

File: docs/docs/guides/faq.mdx [hunk 5]
```diff
@@ -17,6 +17,80 @@ keywords:
...
+
+- **Does Jan support Safetensors?**
+  - At the moment, Jan only supports GGUF. However, there are plans to support `.safetensor` files in the future.
+
+- **I hope to customize the installation path of each model. Is that possible?**
+  - Yes you can customize the installation path. Please see [here](https://github.com/janhq/jan/issues/1010) for more information.
```
Comment (hieu-jan) [inline docs/docs/guides/faq.mdx:85]: Better to point to advanced settings `https://jan.ai/guides/advanced-settings/#access-the-jan-data-folder`

File: docs/docs/guides/integration/groq.mdx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    API integration
+  ]
+---
+
+## How to Integrate Mistral AI with Jan
+
+This guide provides step-by-step instructions on integrating the Groq API with Jan, enabling users to leverage Groq's capabilities within Jan's conversational interface.
+
+Before proceeding, ensure you have the following:
+- Access to the Jan platform
```
Comment (hieu-jan) [inline docs/docs/guides/integration/groq.mdx:25]: Jan application

**Review by hieu-jan — CHANGES_REQUESTED** [review]
There are some issues in current PRs

File: docs/sidebars.js (hunk 1 above)
Comment (cahyosubroto) [reply docs/sidebars.js:159]: Hi @hieu-jan appreciate the feedback! Let's discuss this with @0xSage and @dan-jan in our next meeting

**Review by hieu-jan — APPROVED** [review]
LGTM!

**Pull request merged**